src/test/java/org/folio/innreach/controller/InnReachTransactionControllerTest.java

```diff
@@ -196,6 +200,7 @@ class InnReachTransactionControllerTest extends BaseControllerTest {
   private static final AuditableUser PRE_POPULATED_USER = AuditableUser.SYSTEM;
 
   private static final Duration ASYNC_AWAIT_TIMEOUT = Duration.ofSeconds(15);
+  private static final Duration ASYNC_AWAIT_TIMEOUT_ONE_SECOND = Duration.ofSeconds(1);
```

Comment:
what's the reason to define a new timeout for this specific test?

Reply:
Fixed